sys/net/link_layer/ieee802154/submac.c

```diff
@@ -692,6 +822,9 @@ int ieee802154_submac_init(ieee802154_submac_t *submac, const network_uint16_t *
 
         submac->phy_mode = ieee802154_cap_to_phy_mode(1 << bit);
     }
+    else {
+        submac->phy_mode = IEEE802154_PHY_DISABLED; /* non standard */
```

Comment:
You can just add `IEEE802154_PHY_LORA` if you want to.

Reply:
`ifconfig` showing phy: DISABLED already confused some people.
However something with `IEEE802154_...._LORA` is not really something that I would like to introduce.
I would like that the radio HAL could be used for LoRa, but not that I put LoRa stuff in the IEEE 802154 radio hal, because it is too much experimental.

Bottomline: `IEEE802154_PHY_UNKNOWN` is something I would propose instead.

Reply:
`IEEE802154_PHY_NSTD_LORA` would not make a big difference to me.
If you are certain I will just add `IEEE802154_PHY_LORA`.